newapi/ooniapi/private.py

```diff
@@ -843,3 +841,55 @@ def api_private_circumvention_runtime_stats() -> Response:
 
     except Exception as e:
         return jsonify({"v": 0, "error": str(e)})
+
+
+@api_private_blueprint.route("/domain_metadata")
+def api_private_domain_metadata() -> Response:
+    domain = request.args.get("domain")
```

Comment:
Please, include an explanatory message to this algorithm where:

1. you explain the problem we're trying to solve in terms of (a) canonical category of a domain; (b) managing www (the ideal here would link to the issue we're solving here, which I don't know which is)

2. you refer to the your gist where you investigated the test list

Reply:
Added more doc strings and a link to the gist